contracts/MirrorNode.sol

```diff
@@ -124,6 +124,13 @@ abstract contract MirrorNode {
         return address(uint160(accountNum));
     }
 
+    function accountExist(address account) external returns (bool) {
```

Comment:
I inlined this into `HtsSystemContractJson`, no use case (at least for now) to have it here.